Doc/library/stdtypes.rst

```diff
@@ -1696,10 +1730,27 @@ expression support in the :mod:`re` module).
 .. method:: str.endswith(suffix[, start[, end]])
 
    Return ``True`` if the string ends with the specified *suffix*, otherwise return
-   ``False``.  *suffix* can also be a tuple of suffixes to look for.  With optional
-   *start*, test beginning at that position.  With optional *end*, stop comparing
-   at that position.
+   ``False``::
+
+      >>> 'Python'.endswith('on')
+      True
+
+   *suffix* can also be a tuple of suffixes to look for::
+
+      >>> 'a tuple of suffixes'.endswith(('at', 'in'))
+      False
+
+   With optional *start*, test beginning at that position::
+
+      >>> 'a tuple of suffixes'.endswith(('at', 'es'))
+      True
```

Comment:
This example doesn't use *start*, should this be part of the previous example and a new one added here?

Reply:
I see,
Yes, I will make it part of the previous example